data/categories.txt

```diff
@@ -5039,47 +5040,48 @@ th:จิตบำบัด|นักจิตบำบัด
 tr:Psikoterapi|Psikoterapist
 uk:Психотерапія|Психотерапевт
 vi:Tâm lý trị liệu|nhà trị liệu tâm lý
-zh-Hans:心理治疗|心理治疗师
+zh-Hans:心理治疗师
 zh-Hant:心理治療|心理治療師
 
-healthcare-sample_collection
-en:Sample collection
-ar:أخذ العينات|جمع العينات
-be:Калекцыя ўзораў
-bg:Вземане на проби|Събиране на проби
-cs:Odběr vzorků|Sběr vzorků
-da:Prøvetagning|Prøveudtagning
-de:Probenahme
-el:Δειγματοληψία|Συλλογή δείγματος
-es:Muestreo|Recogida de muestras
-et:Proovivõtmine|Proovide kogumine
-eu:Laginketa|Lagin-bilketa
-fa:نمونه برداری|مجموعه نمونه
-fi:Näytteenotto
-fr:Prélèvements biologiques|Échantillonnage
-he:דְגִימָה|אוסף דוגמאות
-hu:Mintavétel|Mintavételezés
-id:Pengambilan sampel|Pengumpulan sampel
-it:Campionamento|Raccolta di campioni
-ja:サンプリング|サンプル採取
-ko:견본 추출|샘플 수집
-mr:नमुना|नमुना संकलन
-nb:Prøvetaking|Prøvesamling
-nl:Bemonstering|Monstername
-pl:Próbkowanie|Zbieranie próbek
-pt:Amostragem|Recolha de amostras
-pt-BR:Amostragem|Coleta de amostras
-ro:Eșantionare|Colectarea probelor
-ru:Выборка|Сдача анализов
-sk:Vzorkovanie|Zber vzoriek
-sv:Provtagning|Insamling av prover
-sw:Sampuli|Mkusanyiko wa sampuli
-th:การสุ่มตัวอย่าง|เก็บตัวอย่าง
-tr:Örnekleme|Örnek toplama
-uk:Збір аналізів
-vi:lấy mẫu|Bộ sưu tập mẫu
-zh-Hans:抽样调查|样品收集
-zh-Hant:採樣|樣品採集
+# I suspect that "samples" are translated wrong now :)
```

Comment:
Can we leave only a few valid translations here for maps generation, and add other translations later?